src/components/CabinList/index.js

```diff
@@ -1,21 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import cabinData from '../../data/cabin';
 
 function CabinList() {
   const click = (data) => {
+    setCabinName(data);
     console.log(data);
   };
+  const [cabinName, setCabinName] = useState('Select Cabin');
```

Comment:
although what you've done here isn't wrong, it's worth refactoring it slightly.  cabinName should really only ever be a valid name of a cabin, or nothing (an empty string).  'Select Cabin' isn't really a state value in this case, it's a interface element that should be conditionally shown to user if the state is empty.  SO, technically you should show this message based on the state value. 

So - make the default state '' (i.e. an empty string), and then, update your H1 so that if the cabinName state is empty (i.e. it's an empty string) then show the 'Select Cabin', otherwise show the cabinName